layers/+lang/emacs-lisp/packages.el

```diff
@@ -80,7 +80,7 @@
           (lisp-indent-line))))))
 
 (defun emacs-lisp/post-init-company ()
-  (spacemacs|add-company-backends :backends company-capf
+  (spacemacs|add-company-backends :backends (company-capf company-dabbrev-code)
```

Comment:
from `company-diag`, you will find `company-dabbrev-code` already enabled.

Reply:
Really? Did you try my `test-function` example above?
I know that `company-dabbrev-code` is activated by default, but, at least in `emacs-lisp-mode` all completion is `hijacked` by `company-capf`. It is explained in a little more detail in the thread I have linked to (but also the comment itself) above.

Reply:
you're right.
currently `company-capf` and `company-dabbrev` are two separate entities in the `company-backends` and `compnay-dabbrev` is shadowed whenever `company-capf` or any other backend before it returns non-nil value.

grouping `company-capf` and `company-dabbrev` in one list should fix the problem.